pages/price-feeds/price-feeds/price-feed-ids.mdx

```diff
@@ -0,0 +1,37 @@
+import PriceFeedTable from "../../../components/PriceFeedTable";
+import { useState, useEffect } from 'react';
+
+export const PriceFeedData = () => {
+  const [priceFeeds, setPriceFeeds] = useState([]);
+
+useEffect(() => {
+const fetchPriceFeeds = async () => {
+try {
+const response = await fetch('https://hermes.pyth.network/v2/price_feeds');
+const data = await response.json();
```

Comment:
Ideally, you should validate the response type here.  This is where `zod` can be helpful.

Alternatively if you use the Hermes js client, it already has validation and everything built in